modules/screen-create-post/index.js

```diff
@@ -0,0 +1,147 @@
+import React from "react";
+import { Text, View, Image, ScrollView, StyleSheet, TouchableHighlight } from "react-native";
+
+const pressed = () => {
+  console.log("pressed");
+};
+
+export const CreatePost = () => {
```

Comment:
Please change this to be a default export:
https://crowdbotics.github.io/modules/authoring#screen-modules

Reply:
Resolved.